### ISSUE TITLE
"Dismiss" on the game page should do whatever "Next game" would do

### ISSUE THREAD

**Issue by irilyth** [buttonmen-dev/buttonmen#2341]
On https://github.com/buttonmen-dev/buttonmen/issues/1246#issuecomment-53405763 I speculated that "Dismiss on the game page should do whatever Next Game would do, since the thing you want to do after you dismiss a game is to go to your next game, not necessarily go to the overview page." I offered to create a new ticket for that, because I wasn't sure if it would be part of #1246, but it looks like it wasn't, and I never created a new ticket. Here it is!

**Comment by AdmiralJota** [comment]
I just tested this, and when I dismissed a completed game, it *did* automatically bring me to my next game. Do you have the monitor preferences set up for that?

**Comment by irilyth** [comment]
If I have `Automatically Monitor after "Next game" runs out` turned on, it takes me to my next game (via the Monitor). If I turn that off, because I don't want to go into Monitor mode after "Next game" runs out, it instead takes me to the Overview page.

**Comment by irilyth** [comment]
Right, but in this case, I'm playing through my games, and I want to keep playing through my games. I clicked "Next game", took my move, the game finished, I clicked "Dismiss game" -- I want to now go on to my next game, just like would happen if I clicked on "Next game".

From the *Overview* page, if I click "Dismiss game", I think it should stay on the Overview page, just like it does now.

**Comment by AdmiralJota** [comment]
Ah, I hadn't considered the case where you take the final move which finishes the game, dismiss it right there, and don't have the monitor preference set. Interesting.
